packages/sso-login/src/modules/sagas.js

```diff
@@ -22,7 +22,7 @@ export function* loadProviders() {
 }
 
 export function* loginCompleted({payload: {result}}) {
-  yield put(externalEvents.fireExternalEvent('loginCompleted', {result}))
+  yield put(externalEvents.fireExternalEvent('loginCompleted', result))
```

Comment:
Seems like you have created your branch on master. 
Maybe everything will look good if you rebase pr/storybook with master